pkg/publicapi/endpoint/orchestrator/job.go

```diff
@@ -92,7 +92,8 @@ func (e *Endpoint) getJob(c echo.Context) error { //nolint: gocyclo
 			if response.History != nil {
 				continue
 			}
-			history, err := e.store.GetJobHistory(ctx, jobID, jobstore.JobHistoryFilterOptions{})
+			jobHistoryQueryResponse, err := e.store.GetJobHistory(ctx, jobID, jobstore.JobHistoryQuery{})
+			history := jobHistoryQueryResponse.JobHistory
```

Comment:
you should do nil check validation first

Reply:
Done.

Reply:
I think we should check the error before assigning `history := jobHistoryQueryResponse.JobHistory` else this introduce the potential for panics if `GetJobHistory` returns `nil` and an `err`, which is a common practice.